ass1-sass/sass/pages/_home.scss

```diff
@@ -0,0 +1,444 @@
+//scss-lint:disable NestingDepth SelectorDepth
+//============Navbar Start============//
+
+.nav-wrapper {
+  box-shadow: 0 0 2px 2px $light-black;
+  height: 75px;
+  overflow: hidden;
+  position: relative;
+  z-index: 100;
+
+  .navbar {
+    background-color: $white;
+    box-shadow: 0 0 2px 2px $light-black;
+    height: 75px;
+    padding: 0 20px;
+    position: fixed;
+    text-transform: uppercase;
+    width: 100%;
+
+    .brand-logo {
+      float: left;
+
+      img {
+        margin-top: 10px;
+        width: 200px;
+      }
+    }
+
+    .menu {
+      margin: 22px 50px;
+
+      i {
+        font-size: 30px;
+      }
+    }
+
+    .search-container {
+      background-color: $light-brown;
+      border-radius: 3px;
+      box-sizing: border-box;
+      margin: 18px;
+      padding: 5px 10px;
+      width: 50%;
+
+      form {
+        line-height: 30px;
+        margin-bottom: 0;
+
+        input[type=search] {
+          background-color: $light-brown;
+          margin-left: 10px;
+          width: 90%;
+        }
+      }
+    }
+
+    .search-container-mobile {
+      display: none;
+    }
+  }
+}
+
+//============Navbar Ends============//
+
+//=============Sidebar===============//
+
+.sidebar {
+  background-color: $blue;
+  color: $light-grey;
+  height: 100%;
+  overflow: hidden;
+  padding: 20px 0;
+  position: fixed;
+  width: 260px;
+
+  .sidebar-heading {
+    border-bottom: 1px solid $grey;
+    margin-bottom: 15px;
+    padding: 20px;
+
+    div {
```

Comment:
@priyanka0901 Don't add styling for div. What if in future if you add one more div.